weasyprint/layout/column.py

```diff
@@ -176,6 +176,9 @@ def columns_layout(context, box, bottom_space, skip_stack, containing_block,
                 new_boxes.append(new_box)
                 column_skip_stack = resume_at
 
+                # Avoid edge cases resulting in a UnboundLocalError in line 223
```

Comment:
I wouldn't add a comment here. To me it is obvious that there are multiple conditions where `next_box_height` is not set.